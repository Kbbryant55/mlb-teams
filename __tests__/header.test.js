import Header from "../src/components/header";
import { render, screen, within } from "@testing-library/react";
import "@testing-library/jest-dom";

describe("Header", () => {
  it("renders a Header", () => {
    render(<Header />);
    const { getByText } = within(screen.getByTestId("header"));
    expect(getByText("Major League Baseball")).toBeInTheDocument();
  });

  it("renders Header unchanged", () => {


    const { container } = render(<Header />);
    expect(container).toMatchSnapshot();
  });
});
