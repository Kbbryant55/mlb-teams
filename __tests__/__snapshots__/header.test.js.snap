// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders Header unchanged 1`] = `
<div>
  <a
    class="flex pt-3 pb-5 items-center cursor-pointer hover:scale-105 transition ease-in-out"
    href="/"
  >
    <img
      alt=""
      class="mr-2 h-[40px] w-[40px] relative top-[2px]"
      src="https://a.espncdn.com/combiner/i?img=/i/teamlogos/leagues/500/mlb.png&w=64&h=64&scale=crop&cquality=40&location=origin"
    />
    <h1
      class="text-xl font-extrabold"
      data-testid="header"
    >
      Major League Baseball
    </h1>
  </a>
</div>
`;
